projects/components/src/table/table.component.ts

```diff
@@ -55,7 +55,7 @@ import {
     <cdk-table
       [multiTemplateDataRows]="this.isDetailType()"
       [dataSource]="this.dataSource"
-      [ngClass]="[this.display, this.pageable ? 'bottom-margin' : '']"
+      [ngClass]="[this.display, this.pageable && this.display === '${TableStyle.FullPage}' ? 'bottom-margin' : '']"
```

Comment:
probably define a boolean flag the component level and then it can be used at both places? `isTableFullPage`

Reply:
@skjindal93 done